src/components/settings/Settings.h

```diff
@@ -36,6 +36,7 @@ namespace Pinetime {
       };
       enum class PTSGaugeStyle : uint8_t { Full, Half, Numeric };
       enum class PTSWeather : uint8_t { On, Off };
+      enum class DfuAndFsMode : uint8_t { Disabled = 0, Enabled = 1, EnabledTillReboot = 2 };
```

Comment:
I always like to explicitly add enum values (just in case somebody will insert something in the middle of enum and enum is stored in non-volatile memory), but as this is not done in 7 of 8 other enums in this file, I have removed the values.

Reply:
I agree on this one. I guess we rely on the fact that the entire settings file gets wiped if the version changes, so if new options are added it doesn't matter?

Reply:
Yes - increasing `settingsVersion` should wipe any incorrect enum values stored in settings.